manifests/params.pp

```diff
@@ -75,7 +75,7 @@
             },
           }
         }
-        'wheezy', 'jessie', 'precise','quantal','raring','saucy', 'trusty', 'utopic': {
+        'wheezy', 'jessie', 'stretch', 'precise','quantal','raring','saucy', 'trusty', 'utopic': {
           $java =  {
             'jdk' => {
               'package'          => 'openjdk-7-jdk',
```

Comment:
stretch doesn't have java7 packages

Reply:
![](https://media.giphy.com/media/26gsu5MRf7n2WA5uE/giphy.gif)

Reply:
![](https://media.giphy.com/media/26gsu5MRf7n2WA5uE/giphy.gif)

Reply:
welp

Reply:
```
root@lion:~# dpkg -l | grep jdk
ii  openjdk-6-jre:amd64                6b38-1.13.10-1~deb7u1                   amd64        OpenJDK Java runtime, using Hotspot JIT
ii  openjdk-6-jre-headless:amd64       6b38-1.13.10-1~deb7u1                   amd64        OpenJDK Java runtime, using Hotspot JIT (headless)
ii  openjdk-6-jre-lib                  6b38-1.13.10-1~deb7u1                   all          OpenJDK Java runtime (architecture independent libraries)
ii  openjdk-7-jdk:amd64                7u171-2.6.13-1~deb8u1                   amd64        OpenJDK Development Kit (JDK)
ii  openjdk-7-jre:amd64                7u171-2.6.13-1~deb8u1                   amd64        OpenJDK Java runtime, using Hotspot JIT
ii  openjdk-7-jre-headless:amd64       7u171-2.6.13-1~deb8u1                   amd64        OpenJDK Java runtime, using Hotspot JIT (headless)
```

Reply:
that's from a jessie repo, you can tell by the deb8u1. If you get a clean stretch vm there's noo openjdk-7-jdk

Reply:
I don't know that all the vars are going to line up, but might be able to pull some info from https://github.com/puppetlabs/puppetlabs-java/blob/master/manifests/params.pp#L158-L173

Reply:
ahh, okay, I'll run with that! Thanks!